airbyte/cli.py

```diff
@@ -447,6 +453,94 @@ def benchmark(
     )
 
 
+@click.command()
+@click.option(
+    "--source",
+    type=str,
+    help=(
+        "The source name, with an optional version declaration. "
+        "If the name contains a colon (':'), it will be interpreted as a docker image and tag. "
+    ),
+)
+@click.option(
+    "--destination",
+    type=str,
+    help=(
+        "The destination name, with an optional version declaration. "
+        "If a path is provided, it will be interpreted as a path to the local executable. "
+    ),
+)
+@click.option(
+    "--streams",
+    type=str,
+    default="*",
+    help=(
+        "A comma-separated list of stream names to select for reading. If set to '*', all streams "
+        "will be selected. Defaults to '*'."
+    ),
+)
+@click.option(
+    "--Spip-url",
```

Comment:
Goal here with prefixing "S" or "D" to "config" and "pip_url" is to keep the typing down while still being explicit and easy to read.

- `--Sconfig`
- `--Spip_url`
- `--Dconfig`
- `--Dpip_url`

Alternatively, we could just use the longer names:

- `--source-config`
- `--source-pip-url`
- `--destination-config`
- `--destination-pip-url`